doc/cephadm/install.rst

```diff
@@ -121,28 +121,41 @@ curl-based installation
 
      python3.8 ./cephadm <arguments...>
 
-* Although the standalone cephadm is sufficient to get a cluster started, it is
-  convenient to have the ``cephadm`` command installed on the host.  To install
-  the packages that provide the ``cephadm`` command, run the following
-  commands:
+.. _cephadm_update:
 
-  .. prompt:: bash #
-     :substitutions:
+update cephadm
+--------------
 
-     ./cephadm add-repo --release |stable-release|
-     ./cephadm install
+The cephadm binary can be used to bootstrap a cluster and for a variety
```

Comment:
Binary? Thought it was a script.

Reply:
Not any more. It was often referred to as "the binary" prior to the recent changes - but now it is a python [zipapp](https://docs.python.org/3/library/zipapp.html).